modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -0,0 +1,22 @@
+[id="enabling-ang-giving-access-to-the-bulk-import-feature"]
+= Enabling and giving access to the Bulk Import feature
+
+.Procedure
+. To give {product-short} access to GitHub, link:{linkgettingstartedguide}#configuring-github-integration[configure GitHub integration].
+
+. The Bulk Import plugin is installed but disabled by default.
+Enable the `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic` and `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import` preinstalled plugins.
+See link:{plugins-configure-book-url}[{plugins-configure-book-title}].
```

Comment:
There is no docs to explain how to enable this plugin specifically in https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/configuring_dynamic_plugins/index

It might make sense to provide an app-config YAML fragment here. @rm3l Can you provide the required YAML fragment?

Reply:
Yes, that's a good point. I thought we had a generic doc on how to enable a pre-installed plugin..

- So for Helm, that would mean adding this to the values file:
```yaml
global:
  dynamic:
    includes:
      - dynamic-plugins.default.yaml
    plugins:
      ### Bulk Import
      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic
        disabled: false
      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import
        disabled: false
# [...]
```

- And for the Operator, adding this to the dynamic-plugins ConfigMap (and making sure the ConfigMap is referenced in the `spec.application.dynamicPluginsConfigMapName` CR field):
```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: dynamic-plugins-rhdh
data:
  dynamic-plugins.yaml: |
    includes:
      - dynamic-plugins.default.yaml
    plugins:
      # [...]
      # Enable Bulk import plugins.
      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic
        disabled: false
      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import
        disabled: false
```

And (example CR):
```yaml
apiVersion: rhdh.redhat.com/v1alpha1
kind: Backstage
metadata:
  name: my-rhdh
spec:
  application:
    appConfig:
      configMaps:
        - name: app-config-rhdh
    dynamicPluginsConfigMapName: dynamic-plugins-rhdh
  # [...]
```

Reply:
I am documenting here only the content in the your `dynamic-plugins.yaml` file. The rest should be better documented in the plugins book

Reply:
```suggestion
See link:{installing-and-viewing-dynamic-plugins-url}[{installing-and-viewing-dynamic-plugins-title}].
```